chapter_linear-networks/softmax-regression_vn.md

```diff
@@ -186,7 +190,11 @@ Moreover, depending on the inputs, they can take negative values.
 These violate basic axioms of probability presented in :numref:`sec_prob`
 -->
 
-*dịch đoạn phía trên*
+Bạn có thể muốn đề xuất rằng ta có thể lấy kết quả trực tiếp từ hàm logit làm đầu ra để dự đoán.
```

Comment:
```suggestion
Bạn có thể muốn đề xuất rằng ta có thể lấy trực tiếp logit $o$ làm đầu ra mong muốn.
```

Reply:
@lkhphuc chỗ này thì em nghĩ logit $o$ là một hàm toán học thì ta chỉ có thể lấy kết quả của hàm đó thôi, chứ không thể lấy nguyên hàm đó ra ạ.

Reply:
Trong Stats đúng là `logits` là hàm nghịch của softmax thật, nhưng theo mình hiểu trong ngữ cảnh ML mình thường hay xài `logit` đơn giản để chỉ đầu vào của hàm `softmax` thôi à. Không biết hiểu đúng không?